website/docs/plugin/testing/migrating.mdx

```diff
@@ -0,0 +1,33 @@
+---
+page_title: 'Plugin Development: Migrating testing from SDKv2 to the testing module'
+description: >-
+Migrate your provider's acceptance testing dependencies from SDKv2 to the testing module.
+---
+
+# Overview
+
+This guide helps you migrate a Terraform provider's acceptance testing dependencies from SDKv2 to the plugin testing module. We recommend migrating to terraform-plugin-testing to take advantage of new features of the testing module and to avoid importing the SDKv2 for providers that are built on the plugin Framework.
+
+This guide provides information and examples for most common use cases, but it does not discuss every nuance of migration. You can ask additional migration questions in the [HashiCorp Discuss forum](https://discuss.hashicorp.com/c/terraform-providers/tf-plugin-sdk/43). To request additions or updates to this guide, submit issues or pull requests to the [`terraform-plugin-testing` repository](https://github.com/hashicorp/terraform-plugin-testing).
+
+
+## Migration steps
+
+Take the following steps when you migrate a provider's acceptance tests from SDKv2 to the testing module:
+
+- Change all instances of the following Go import statements in `*_test.go` files:
+    - `github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest` to `github.com/hashicorp/terraform-plugin-testing/helper/acctest`
+    - `github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource` to `github.com/hashicorp/terraform-plugin-testing/helper/resource`
```

Comment:
Some folks may be tangentially pulling in the `terraform` package as well (e.g. for custom `TestCheckFunc`), so it'll be good to give that a shout as well:

```suggestion
    - `github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource` to `github.com/hashicorp/terraform-plugin-testing/helper/resource`
    - `github.com/hashicorp/terraform-plugin-sdk/v2/terraform` to `github.com/hashicorp/terraform-plugin-testing/terraform`
```